### ISSUE TITLE
[Bug] grpo_loss_with_old_logps double-negates the loss (wrong sign)

### ISSUE THREAD

**Issue by truong-v** [fla-org/flash-linear-attention#1073]
`grpo_loss_with_old_logps` in `fla/modules/grpo.py` negates the loss twice, so the returned loss has the wrong sign.

```python
# fla/modules/grpo.py:415-419
token_loss = -(torch.min(advantages * importance_weights, advantages *
               importance_weights_clipped) - beta * per_token_kl) * completion_mask

loss = -token_loss.sum() / completion_mask.sum()
```

`token_loss` is already the per-token GRPO loss; the second `-` flips it back, giving `loss = +(min(A*w, A*w_clip) - beta*KL)` averaged over valid tokens. Minimizing that pushes the policy away from high-advantage actions and increases KL to the reference.

`grpo_loss_torch` (line 351) and the TRL snippet in the module docstring (line 45) both apply the negation only once.

Fix:

```python
loss = token_loss.sum() / completion_mask.sum()
```

The function is currently unused and untested (not exported from `fla/modules/__init__.py`, no references elsewhere), so nothing is affected today.

Two smaller issues in the same function:

- Line 405: `torch.arange(logits_to_keep)[None, :] >= 0` is `True` everywhere, so the mask reduces to `pad_mask` and `logits_to_keep` has no effect.
- Line 390: `rewards.view(-1, B)` with `B = logps.shape[0]` and `rewards` of shape `[B]` yields `(1, B)`, so the advantage mean/std are taken over the whole batch rather than per-prompt generation groups — despite the `# B,num_generations` comment.

Happy to send a PR if useful.
